firmware/.gitignore

```diff
@@ -0,0 +1,5 @@
+.pio
+.vscode/.browse.c_cpp.db*
+.vscode/c_cpp_properties.json
+.vscode/launch.json
+.vscode/ipch
```

Comment:
Please don't commit this `firmware/.gitignore` file. You can just delete the file locally, all exclusions you are adding are already tracked in the repository's root `.gitignore`.

Reply:
Ok daniel